.github/workflows/system-tests.yml

```diff
@@ -300,3 +300,14 @@ jobs:
           name: logs_parametric
           path: artifact.tar.gz
 
+  finished:
+    runs-on: ubuntu-latest
+    needs: [parametric, system-tests]
+    if: success() || failure()
```

Comment:
Is this the same as:
```
if: always()
```
?

I wonder if `cancelled` is some kind of different state:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#cancelled

Reply:
I can't remember exactly why I did this, I think I just didn't care since cancelled means we aren't going to finish anyways